### PR TITLE
Rename ScheduledJobs to CronJobs

### PR DESCRIPTION
**Pull request by soltysh**
This builds on top of #1506, since the other wasn't merged yet and it also includes some changes that needs to be reflected here. 

@kubernetes/docs ptal

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1651)

### CODE REVIEW

**Comment by soltysh** [comment]
This one will also fix travis, due to the ScheduledJob->CronJob rename thing, so it can replace the other PR as well.

**Comment by kargakis** [comment]
> This one will also fix travis, due to the ScheduledJob->CronJob rename thing, so it can replace the other PR as well.

Came here for that, cool

**Comment by janetkuo** [comment]
LGTM, but needs docs review from @kubernetes/docs, specifically:

1. Does the new navigation look good -- (1) `Guides > Batch Jobs > Cron Jobs` and (2) `Reference > Glossary > Cron Jobs`
2. If you click on (2), you'll be brought to (1). Is that acceptable?

File: _data/reference.yml [hunk 1]
```diff
@@ -183,45 +183,45 @@ toc:
...
   - title: Replication Controller
-    path: /docs/user-guide/replication-controller/    
+    path: /docs/user-guide/replication-controller/
   - title: Resource Quotas
-    path: /docs/admin/resource-quota/    
-  - title: Scheduled Jobs
-    path: /docs/user-guide/scheduled-jobs/
+    path: /docs/admin/resource-quota/
+  - title: Cron Jobs
+    path: /docs/user-guide/cron-jobs/
```
Comment (janetkuo) [inline _data/reference.yml]: Clicking on this brings you back to `Guildes` page

Comment (soltysh) [inline _data/reference.yml]: The same applies to Jobs and few other resources, I haven't changed anything other than just renaming resources, here :wink:

Comment (janetkuo) [inline _data/reference.yml]: Yes but the change in `_data/guides.yml` introduced this behavior

Comment (soltysh) [inline _data/reference.yml]: Not quite, this PR builds on top of #1506 which added ScheduledJob to `_data/guides.yml`. Previously it wasn't even available there. I've just checked against what we have in master going through `Reference > Glossary > Jobs` will get you to `Guides > Batch Jobs > Jobs`. Not sure what would you like to see there?

Comment (janetkuo) [inline _data/reference.yml]: > I've just checked against what we have in master going through Reference > Glossary > Jobs will get you to Guides > Batch Jobs > Jobs.

IMO the navigation experience may be confusing. Not sure if it's intentional and if we want to keep that or not.

Comment (soltysh) [inline _data/reference.yml]: I'll leave that decision to @kubernetes/docs

**Comment by soltysh** [comment]
Rebased @kubernetes/docs ptal

File: _data/reference.yml (hunk 1 above)
Comment (devin-donnelly) [inline _data/reference.yml]: We don't want to keep it, but a lot of links from Reference/Glossary go to places in Guides.

That will change as we build out Concepts. This is fine for now.

**Comment by devin-donnelly** [comment]
Can you fix the merge conflict?

**Comment by soltysh** [comment]
Rebased, will merge on green.

**Pull request merged**